python-package/xgboost/dask.py

```diff
@@ -771,11 +774,9 @@ async def _predict_async(client, model, data, missing=numpy.nan, **kwargs):
         raise TypeError(_expect([DaskDMatrix, da.Array, dd.DataFrame],
                                 type(data)))
 
-    _global_config = config.get_config()
-
```

Comment:
What is wrong with this line? Does it receive the config of the worker, not the scheduler? Why did the unit test not catch the problem?